### PR TITLE
Update ZDV URL to new site

### PR DESCRIPTION
**Pull request by Celeo**
Unlike the other PRs to the other repos, this PR includes changes to the deployment process, which should probably be checked by an extra-careful eye. :)

### CODE REVIEW

**Review by dhawton — CHANGES_REQUESTED** [review]
PRs should not have creep. There's a lot in here that is not relevant to the proposed change. This makes it hard to review and comment on. Each PR should have an individual thing you're changing, whether that's a bug, feature, etc.

**Comment by Celeo** [comment]
Removed formatting of `docs/swagger.json`; PR is 24 LOCC.

**Review by dhawton — CHANGES_REQUESTED** [review]
PR still has creep

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -79,7 +79,7 @@ jobs:
       run: cosign sign --key env://COSIGN_PRIVATE_KEY ${{ env.image_digest_sha }} ${{ env.image_digest_branch }} -y
       env:
         COSIGN_PASSWORD: ${{ secrets.COSIGN_PRIVATE_KEY_PASSWORD }}
-        COSIGN_PRIVATE_KEY: ${{ secrets.COSIGN_PRIVATE_KEY }}
```
Comment (dhawton) [inline .github/workflows/build.yaml:82]: While this change is nice, this is creep. It is not relevant to updating the ZDV URL.

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -109,7 +109,7 @@ jobs:
       run: |-
         cd $GITHUB_WORKSPACE/gitops
         git config user.name "${{ secrets.GH_PIPELINE_USERNAME }}"
-        git config user.email "wm@denartcc.org"
+        git config user.email "wm@zdvartcc.org"
```
Comment (dhawton) [inline .github/workflows/build.yaml:112]: This change isn't completely necessary.

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ ADH Partnership Monolithic API.
 
 ## Introduction
 
-This monolithic API is meant to break apart the previous Laravel web application into a backend and frontend.  This backend will provide data to the frontend as well as other web components in the future, ie: IDS.
+This monolithic API is meant to break apart the previous Laravel web application into a backend and frontend. This backend will provide data to the frontend as well as other web components in the future, ie: IDS.
```
Comment (dhawton) [inline README.md:7]: This change is unnecessary, and is not related to changing domains.

File: README.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ The API follows standard RESTful API designs. The API documentation is accessibl
 
 ## Tests
 
-Tests are a good thing and should be written where practical.  To run tests:
+Tests are a good thing and should be written where practical. To run tests:
```
Comment (dhawton) [inline README.md:21]: See above.

File: README.md [hunk 5]
```diff
@@ -33,7 +33,7 @@ There are a few ways to use this... the preferred is using a Kubernetes deployme
 
 ### Hard Coded Discord Webhook Names
 
-These should be configured in the config.yaml file with appropriate webhook URLs.  The webhook names are:
+These should be configured in the config.yaml file with appropriate webhook URLs. The webhook names are:
```
Comment (dhawton) [inline README.md:36]: Same here

File: README.md [hunk 6]
```diff
@@ -42,7 +42,7 @@ These should be configured in the config.yaml file with appropriate webhook URLs
 - pending_feedback - Notification about new feedback submissions
 - broadcast_feedback - Approved feedback reports are broadcast to this webhook, intended to be public praises
 - role - Notifications about added/removed roles
-- visitng_application - Notifications related to Visiting Applications (new, approved, denied, etc), should be restricted
+- visiting_application - Notifications related to Visiting Applications (new, approved, denied, etc), should be restricted
```
Comment (dhawton) [inline README.md:45]: While a bug, this should be a separate PR

File: cmd/api/main.go [hunk 7]
```diff
@@ -26,7 +26,7 @@ func main() {
 // @license.name Apache
 // @license.URL https://github.com/adh-partnership/api/blob/main/LICENSE
 
-// @host network.denartcc.org
+// @host network.zdvartcc.org
```
Comment (dhawton) [inline cmd/api/main.go:29]: This is unused, so shouldn't be changed

File: config.yaml.example [hunk 8]
```diff
@@ -70,14 +70,14 @@ storage:
   region: {{.STORAGE_REGION | default "us-east-1"}}
   endpoint: {{.STORAGE_ENDPOINT | default ""}}
 oauth:
-  base_URL: "{{.OAUTH_BASEURL | default "https://auth.denartcc.org" }}"
+  base_URL: "{{.OAUTH_BASEURL | default "https://auth.zdvartcc.org" }}"
```
Comment (dhawton) [inline config.yaml.example:73]: Do not change examples. This was not updated because I don't want working examples included.

File: internal/v1/storage/utils.go [hunk 9]
```diff
@@ -1,6 +1,6 @@
 package storage
 
-var base = "https://cdn.denartcc.org/"
+var base = "https://cdn.zdvartcc.org/"
```
Comment (dhawton) [inline internal/v1/storage/utils.go:3]: This change is unnecessary, as the value is fed in through configuration. Ideally this should be refactored out since this API isn't just for ZDV any longer, but is replaced in main.go.

File: manifest/configmap.yaml [hunk 10]
```diff
@@ -25,7 +25,7 @@ data:
         name: "zdv_session"
         key: "{{.SESSION_KEY | default "zdv_session"}}"
         max_age: 604800
-        domain: "{{.SESSION_DOMAIN | default ".denartcc.org"}}"
```
Comment (dhawton) [inline manifest/configmap.yaml:28]: Same comment about modifying examples.
